CHANGELOG.md

```diff
@@ -3,6 +3,78 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.15.1
+_Sep 28, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🧰 Fix header filters menu auto closing on render (#10483) @MBilalShafi
+- 🌳 Do not try to focus a collapsed node when re-focusing the TreeView (#10422) @flaviendelangle
+- 🎯 Fix column headers scroll when theme scoping is used (#10437) @cherniavskii
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@6.15.1`
+
+- [DataGrid] Fix column headers scroll when theme scoping is used (#10437) @cherniavskii
+- [DataGrid] Rename `global` to `globalScope` due to Jest issue (#10470) @romgrk
+- [l10n] Improve Russian (ru-RU) locale (#10464) @NKodos
+- [l10n] Improve Russian (ru-RU) locale on the DataGrid (#10407) @NKodos
```

Comment:
```suggestion
- [l10n] Improve Russian (ru-RU) locale (#10407) @NKodos
```

Reply:
I did group it with the item above and just added the second PR to the parantheses. Is that ok?